vars/buildDockerAndPublishImage.groovy

```diff
@@ -62,7 +62,7 @@ def call(String imageName, Map userConfig=[:]) {
               # Add folder to $PATH
               $env:Path += "$env:WORKSPACE\\.bin"
               # Add "tools" folder to $PATH
-              $env:Path += "C:\\tools"
+              $env:Path += ";C:\\tools"
```

Comment:
I might be mistaken but:

https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#withenv-set-environment-variables specifies that you can control the Path using the `withEnv()` pipeline step:

```groovy
withEnv(['PATH+TOOLS=C:/tools']) {
  powershell 'echo $env:Path'
}
```

> You may also use the syntax PATH+WHATEVER=/something to prepend /something to $PATH.

Reply:
Also, does this mean that C:\Tools is not in the current path 😱 ?